src/pslLint/elementsConventionChecker.ts

```diff
@@ -25,6 +25,27 @@ export class PropertyStartsWithZ implements PropertyRule {
 	}
 }
 
+export class PropertyIsDummy implements PropertyRule {
+
+	ruleName = PropertyIsDummy.name;
+
+	report(_parsedDocument: PslDocument, property: Property): Diagnostic[] {
```

Comment:
The name `_parsedDocument` implies it is not being used, but here in this method it is. Please remove the `_`.